tools/splat.yaml

```diff
@@ -9,6 +9,7 @@ options:
   extensions: splat_ext
   symbol_addrs_path: tools/symbol_addrs.txt
   platform: n64
+  assets_dir: assets
```

Comment:
I thought assets was the default dir?

Reply:
The default behaviour is to not do assets_dir stuff at all, i.e. splat is backwards-compatible